tidysic/main.py

```diff
@@ -26,10 +26,10 @@
     help='Move non-audio files along with their audio neighbor files.'
 )
 @click.option(
-    '-g',
-    '--guess',
+    '-i',
+    '--interactive',
     is_flag=True,
-    help='Guess the audio file title and artist when there is no IDE tags.'
+    help='Will prompt user input for missing tags.'
```

Comment:
I would keep the present tense here, as we did for every other help message:

"Prompt user input for missing tags."